src/test/scala-2.11/markets/participants/OrderCancelerSpec.scala

```diff
@@ -52,35 +52,59 @@ class OrderCancelerSpec extends TestKit(ActorSystem("OrderCancelerSpec"))
 
   feature("A OrderCanceler should be able to schedule SubmitOrderCancellation messages.") {
 
-    val market = TestProbe()
-    val prng = new Random(42)
-    val ticker = Agent(Tick(1, 1, Some(1), 1, 1))
     val tradable = TestTradable("GOOG")
+    val market = TestProbe()
+    val markets = mutable.Map[Tradable, ActorRef](tradable -> market.ref)
+    val initialTick = Tick(1L, 1L, Some(1L), 1L, 1L)
+    val tickers = mutable.Map[Tradable, Agent[Tick]](tradable -> Agent(initialTick))
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [It's preferred to use immutable collections instead of mutable ones](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1705065396/issues/source?bid=2921361&fileBranchId=3046267#l59)